conf/messages.en

```diff
@@ -466,7 +466,7 @@ guaranteeDetails.addAnotherGuarantee.singular.title = You have added {0} guarant
 guaranteeDetails.addAnotherGuarantee.singular.heading = You have added {0} guarantee
 guaranteeDetails.addAnotherGuarantee.plural.title = You have added {0} guarantees
 guaranteeDetails.addAnotherGuarantee.plural.heading = You have added {0} guarantees
-guaranteeDetails.addAnotherGuarantee.error.required = Select yes if you need to add another guarantee
+guaranteeDetails.addAnotherGuarantee.error.required = Select yes if you want to add another guarantee
```

Comment:
The below message should probably be changed from "need" to "want" as well in that case

Reply:
:+1: